src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Build/Diagnostics/IncrementalBuildFailureDetector.cs

```diff
@@ -127,14 +127,9 @@ int IVsUpdateSolutionEvents2.UpdateProjectCfg_Done(IVsHierarchy pHierProj, IVsCf
 
             static BuildAction GetBuildActionFromUpToDateOptions(uint options)
             {
-                if ((options & VSConstants.VSUTDCF_PACKAGE) == VSConstants.VSUTDCF_PACKAGE)
+                if ((options & 0x00100000) == 0x00100000)
```

Comment:
Why no longer use constants?

Reply:
I need to find the right constant here, probably by digging through the VS code. I believe I will revisit this code in time to handle other cases. The previous code was not working correctly anyway.

Reply:
Seems `VSSOLNBUILDUPDATEFLAGS.SBF_*` has the constants we need here.